2023-06-07 01:07:38,957 TADA INFO starting test `ldmsd_auth_test`
2023-06-07 01:07:38,957 TADA INFO   test-id: 4669a6a992a622208e0a7c207a1879352bfb6030f0a99b761c438bb2c185af30
2023-06-07 01:07:38,957 TADA INFO   test-suite: LDMSD
2023-06-07 01:07:38,957 TADA INFO   test-name: ldmsd_auth_test
2023-06-07 01:07:38,958 TADA INFO   test-user: narate
2023-06-07 01:07:38,958 TADA INFO   commit-id: d383cb25cca1600a39c379002728a15a57bfe513
2023-06-07 01:07:38,958 __main__ INFO -- Get or create the cluster --
2023-06-07 01:07:57,106 __main__ INFO -- Start daemons --
2023-06-07 01:08:16,195 __main__ INFO ... wait a bit to make sure ldmsd's are up
2023-06-07 01:08:21,332 TADA INFO assertion 1, root@agg-2(dom3) ldms_ls to agg-2:10000: see all sets, passed
2023-06-07 01:08:21,452 TADA INFO assertion 2, user@agg-2(dom3) ldms_ls to agg-2:10000: see only meminfo, passed
2023-06-07 01:08:21,586 TADA INFO assertion 3, root@headnode(dom4) ldms_ls to agg-2:10001: see all sets, passed
2023-06-07 01:08:21,702 TADA INFO assertion 4, user@headnode(dom4) ldms_ls to agg-2:10001: see only meminfo, passed
2023-06-07 01:08:21,816 TADA INFO assertion 5, root@headnode(dom4) ldms_ls to agg-11:10000: unexpected result: ['Wed Jun 07 01:08:21 2023:    ERROR: auth_munge: munge_decode() failed. Invalid credential', 'Connection failed/rejected.'], failed
2023-06-07 01:08:21,816 TADA INFO test ldmsd_auth_test ended
